src/platform/backends/lxd/lxd_virtual_machine.cpp

```diff
@@ -407,9 +407,13 @@ const QUrl mp::LXDVirtualMachine::network_leases_url()
     return base_url.toString() + "/networks/" + bridge_name + "/leases";
 }
 
-void mp::LXDVirtualMachine::request_state(const QString& new_state)
+void mp::LXDVirtualMachine::request_state(const QString& new_state, const QJsonObject args)
```

Comment:
Couldn't the new param be a const reference instead? Otherwise, move contents in the implementation perhaps? Not sure if that's supported in (our version of) QJson stuff.

Reply:
Nice catch, I overlooked this when I took over the code. The calling code only passed temporary objects into the `request_state` function. Likely, that is why Chris made a const value. But indeed, but general use case, it is better to have it as a const ref, I will change it.